docs/core/deploying/single-file.md

```diff
@@ -34,6 +56,22 @@ For example, to place some files in the publish directory but not bundle them in
 </ItemGroup>
 ```
 
+## Include PDB files inside the bundle
+
+The PDB file for an assembly can be bundled with single-file by setting the following property:
```

Comment:
```suggestion
The PDB file for an assembly can be embedded into the assembly itself (the `.dll`) using the setting below. Since the symbols are part of the assembly, they will be part of the single-file application as well.
```

Probably needs better wording - but basically: embedding symbols does not "bundle them with single-file" - or not directly. I think in the context of this document it would be rather confusing.

Also - do we have some docs on PDB embedding - I would assume so. Maybe link to it.

Reply:
Couldn't quite find a relevant doc related to this but changed wording to reflect the feedback